### PR TITLE
Update C++ example in README.md

### PR DESCRIPTION
**Pull request by dkelson**
Update the C++ example in the README.md so that it will compile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,23 +98,28 @@ wss.on('connection', function (ws) {
 ### C++
 For maximum performance and memory scaling the native interface is recommended. Look in the examples folder for threading and load balancing examples. There is no documentation written yet but a bright person like you will have no problem just reading the header file.
 ```c++
+
+using namespace uWS;
+
```
Comment (alexhultman) [inline README.md]: Why this?

File: README.md [hunk 2]
```diff
@@ -98,23 +98,28 @@ wss.on('connection', function (ws) {
...
 int main()
 {
     /* this is an echo server that properly passes every supported Autobahn test */
-    uWS::Server server(3000);
-    server.onConnection([](uWS::WebSocket socket) {
-        cout << "[Connection] clients: " << ++connections << endl;
+    int connections = 0;
+    uWS::EventSystem es(MASTER);
```
Comment (alexhultman) [inline README.md]: If you prefix MASTER

**Comment by alexhultman** [comment]
It's appreciated but I would rather remove the using namespace uWS given that everything except for MASTER is prefixed with uWS.

File: README.md (hunk 2 above)
Comment (dkelson) [inline README.md]: OK

**Comment by dkelson** [comment]
I'll adjust

**Comment by dkelson** [comment]
how about now?

**Comment by alexhultman** [comment]
thanks, appreciated. I see I had some mixed usage of std:: and not

**Pull request merged**